### PR TITLE
fix the README

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains

* Doing some changes in README 
* Adding Prosebot (Automated Grammar/Spell and consistency checks)

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc : @huan

**Comment by Ayush7614** [comment]
This PR contains some typos

**Comment by Ayush7614** [comment]
This PR contains some typos 
image

**Comment by Ayush7614** [comment]
cc : @huan

**Comment by huan** [comment]
Thanks for creating the PR.

However, could you please explain which "typo" had been fixed in this PR?

And I do not think the description added by you is precious:

> This repo provides information about the organisation Wechaty and its tools. You can find out who we are, what we do and most importantly, how we work and how you can contribute.  

Which "tools" are you talking about above?

**Comment by Ayush7614** [comment]
> Thanks for creating the PR.
> 
> However, could you please explain which "typo" had been fixed in this PR?
> 
> And I do not think the description added by you is precious:
> 
> > This repo provides information about the organisation Wechaty and its tools. You can find out who we are, what we do and most importantly, how we work and how you can contribute.
> 
> Which "tools" are you talking about above?

cc @huan now you can review this i am correct all the things

**Comment by huan** [comment]
Could you please explain "what you have corrected" in the comment and let me know what is "all the things"?

Thanks.

File: .github/write-good.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Prosebot Integration; Turn to False to disable any of these checks.
```
Comment (huan) [inline .github/write-good.yml:1]: I do not know anything about this file.

Please add an explanation by filing an issue for discussion.

File: README.md [hunk 2]
```diff
@@ -5,7 +5,11 @@
 [![TypeScript](https://img.shields.io/badge/%3C%2F%3E-TypeScript-blue.svg)](https://www.typescriptlang.org/)
 [![ES Modules](https://img.shields.io/badge/ES-Modules-brightgreen)](https://github.com/Chatie/tsconfig/issues/16)
 
-## About Wechaty
```
Comment (huan) [inline README.md:8]: Could you please explain why the `About Wechaty` should be replaced by `Wechaty`?

I can not understand it.

File: .github/write-good.yml (hunk 1 above)
Comment (Ayush7614) [reply .github/write-good.yml:1]: Hy @huan Firstly Thanks for reviewing my PR
Ok so this is Prosebot or we can also say this is Github Action(GHA)
* Check for grammatical errors in the docs
* Analyzes spelling, paragraphs, in Markdown files and Text
* Provide an easy way to analyze documentation

**Comment by huan** [comment]
Thanks for your purpose.

However, I do not think they are necessary for us, do I will close this PR.

Please file an issue and describe what problem you meet and try to solve in details first for discussion, and creating PR after there's any agreements.

**Pull request closed without merging**